xdsl/printer.py

```diff
@@ -676,6 +677,12 @@ def print_int_or_question(value: IntAttr | NoneAttr) -> None:
             self.print_string(f"affine_set<{attribute.data}>")
             return
 
+        if isinstance(attribute, TupleType):
+            self.print_string("tuple<", indent=0)
```

Comment:
what's with the `indent`?

Reply:
It bypasses the check for adding indentation for nested printing, we currently do that by printing newlines between things, and adding an indent of 0 fast-tracks to printing the raw string.

Reply:
I do it becasue of a perf measurement I did a while back, but I see that the `print_string` got rewritten and this is now not useful, I'll remove it.